src/renderer/components/GridRender.tsx

```diff
@@ -74,6 +81,10 @@ export default function GridRender() {
                                                 temporalEpoch={epoch}
                                                 key={node.uid}
                                                 connectedEth={clusterChip?.connectedChipsByEthId.get(node.uid) || null}
+                                                shouldRenderLinkSaturation={shouldShowLinkSaturation}
```

Comment:
nit: i believe word `should` doesnt add any value. `shouldRenderLinkSaturation` might as well be `renderLinkSaturation`. same for the rest